erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -755,7 +767,7 @@ def get_sub_assembly_items(bom_no, bom_data):
 			get_sub_assembly_items(bom_item.get("bom_no"), bom_data)
 
 @frappe.whitelist()
-def update_per_received_and_status_in_production_plan(purchase_receipt):
+def update_per_received_in_production_plan(purchase_receipt):
```

Comment:
This is a function called from the frontend. Docstrings should be in format

Reply:
Please correct the details in the docstring - it doesn't reflect what the function is doing